### ISSUE TITLE
Multiple users with different permissions

### ISSUE THREAD

**Issue by AddAndy** [causefx/iDashboard-PHP#13]
It would be nice if there was the ability to have more than 1 user/groups with different permissions.

So myself (admin) would have access to everything, my housemates would have access to most services, and remote users would have read-only access to most of the info.

**Comment by causefx** [comment]
I did this exact thing on a fork from Muximux...  

https://github.com/causefx/MMHTPC-ManageMyHTPC

Not sure if I should do something similar with this.

**Comment by derrickmehaffy** [comment]
I would also like to see this if possible. As I allow my media pc for friends and family I would like to add in a way for them to request movies/tv shows using another tool and would like to incorporate it into the dashboard

**Comment by causefx** [comment]
@AddAndy & @derrickmehaffy 

Not sure if this will help you but you could possibly run two instances of iDashboard like they way I'm doing right now to have nested tabs with different passwords.  The only problems is if there are 2 different instances with 2 different passwords, it bugs out when you refresh the main page and you have to log back in, I could code it so it is more unique to each instance or I could just revisit the whole different users thing.

![image](https://cloud.githubusercontent.com/assets/16184466/19747942/992f8d88-9b93-11e6-8f7d-348c493da763.png)

**Comment by derrickmehaffy** [comment]
That's not a bad idea, slightly more complicated but not too difficult to do. Just curious with things like your Sonarr and what not, do you have authentication enabled on those as well as the main iDashboard? I'm under the assumption your running all those sites on localhost and using the idashboard to iframe them.

I have not yet installed the dashboard and was going to do so right now. :)

**Comment by causefx** [comment]
Correct, I have all of my services running on my server and they are all behind a reverse proxy in nginx.
